### DISCUSSION TITLE
til-site/posts/i-m-turning-30-so-i-m-switching-to-openbsd/

### DISCUSSION THREAD

**Discussion by giscus** [hiAndrewQuinn/til#12] [Announcements]
# til-site/posts/i-m-turning-30-so-i-m-switching-to-openbsd/

I’m kidding, I’m switching to OpenBSD because I like security or code quality or something.
It’s totally not because the inexorable march of aging is starting to show its effects on my ability to down necessary-evil trivia like me and my friends used to down forties in the Ahhhnald after dark, and so I’d like to settle down with a software ecosystem I can study in real depth once without feeling like 20% of what I absorb in year X will be deprecated by year X+10.

https://hiandrewquinn.github.io/til-site/posts/i-m-turning-30-so-i-m-switching-to-openbsd/

**Comment by edwinjhlee** [comment]
Lucky you. I am turning 35 but still don't get time for BSD ...
Take a look at: [x-cmd](https://x-cmd.com)
X-cmd and I will embrace BSD and ksh in the near future.

**Reply by hiAndrewQuinn** [reply]
Looks cool, the splash page could probably use a little more description for what it's about though. I originally thought it was about integrating AI directly into your shell, [kind of like switching to editor mode to get Copilot in the shell](https://andrew-quinn.me/ctrl-x-ctrl-e/), but it sounds much broader than that.

**Reply by hiAndrewQuinn** [reply]
(EDIT: Less constructive criticism than I thought! [NoScript](https://noscript.net/) was stripping out the terminal demo in the rigtht 60%-ish of the screen for me. Confirms again that this looks cool!)
